src/applications/registry.json

```diff
@@ -923,5 +923,21 @@
       "vagovprod": false,
       "layout": "page-react.html"
     }
+  },
+  {
+    "appName": "coronavirus-vaccination",
+    "entryName": "coronavirus-vaccination",
+    "rootUrl": "/coronavirus-vaccination",
+    "template": {
+      "vagovprod": false,
+      "layout": "page-react.html",
+      "includeBreadcrumbs": true,
+      "breadcrumbs_override": [
+        {
+          "path": "/coronavirus-vaccination",
+          "name": "Request a COVID-19 vaccination"
+        }
+      ]
+    }
```

Comment:
I chose "coronavirus-vaccination" because I was using the `/coronavirus-vaccine/` app on the `coronavirus-vaccine-registry-initial` branch as reference in the beginning